tests/runtime/watchpoint

```diff
@@ -1,7 +1,8 @@
 NAME watchpoint - absolute address
-RUN {{BPFTRACE}} -e 'watchpoint:0x10000000:8:w { printf("hit!\n"); exit() }' -c ./testprogs/watchpoint
+RUN install -m 644 /dev/null /tmp/watchpoint_mem; ./testprogs/watchpoint & > /dev/null; {{BPFTRACE}} -e "watchpoint:$(awk '{print $1}' /tmp/watchpoint_mem):8:w { printf(\"hit!\n\"); exit() }"
 EXPECT hit!
 ARCH aarch64|ppc64|ppc64le|x86_64
+AFTER /bin/bash -c "rm -f /tmp/watchpoint_mem"
```

Comment:
Unfortunately the `AFTER` directive does not do what you think it does here. It's not a cleanup directive. Rather, it runs right after the `RUN` command is spawned. So using it like this is racy. Feel free to add a `CLEANUP` directive to the test runner -- shouldn't be difficult.